website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,183 @@
+
+When you run a build job, any saved queries downstream of the dbt models in that job will also run. To make sure your export data is up-to-date, run the export as a downstream step (after the model). 
+
+To create an export: 
+
+1. Create a [deploy job](/docs/deploy/deploy-jobs) in dbt Cloud.
+2.  Set the environment variable `DBT_INCLUDE_SAVED_QUERY=TRUE` in your environment to run any Export that needs to be refreshed after a model is build. 
+   - You can use the [selector syntax](/reference/node-selection/syntax) `--select` to only run the exports downstream of a particular model.
+3. After dbt finishes building the models, the MetricFlow Server processes the exports, compiles the necessary SQL, and executes this SQL against your data platform.
+4. Review the exports execution details in the jobs logs and confirm the export was run successfully. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the job logs.
+5. Your data is now available in the data platform for querying.
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/deploy_exports.jpg" width="90%" title="Adding --include-saved-query to the dbt build command in your job execution settings." />
+
+You can use the selector syntax `--select` or `-s` to specify a particular dbt model to run in your build command in order to only run the exports downstream of that model, or to select a saved query to run. As an example, the following command will run any saved queries that are downstream of the `orders` semantic model:
+
+```bash
+dbt build --select orders+
+```
+
+## FAQs
+
+<detailsToggle alt_header="Can I have multiple exports in a single saved query?">
+
+Yes, this is possible. However, the difference would be the name, schema, and materialization strategy of the export.
+</detailsToggle>
+
+<detailsToggle alt_header="How do I run all exports for a saved query?">
+
+- In production runs, you can build the saved query by calling it directly in the build command, or you build a model and any exports downstream of that model
+- In development, you can run all exports by running `dbt sl export --saved-query sq_name`.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Will I run duplicate exports if multiple models are downstream of my saved query?">
+
+dbt will only run each export once even if it builds multiple models that are downstream of the saved query. For example, you may have a saved query called order_metrics, which has metrics from both the orders and order_items semantic models.
+
+You can run a job that includes both models: `dbt build`. This runs both the `orders` and `order_items` models, however it will only run the `order_metrics` export once.
```

Comment:
```suggestion
You can run a job that includes both models using `dbt build`. This runs both the `orders` and `order_items` models, however, it will only run the `order_metrics` export once.
```